### DISCUSSION TITLE
Review setting per language

### DISCUSSION THREAD

**Discussion by TheLastProject** [WeblateOrg/weblate#8387] [Q&A] [answered]
Hey everyone,

I noticed the section on [Dedicated Reviewers](https://docs.weblate.org/en/latest/workflows.html#dedicated-reviewers), but unless I misunderstood it it seems rather global. And I was wondering:
1. is there any way to set up only some languages to have reviewers, to not add friction to languages for which nobody is available to review anything? Alternatively or additionally,
2. is there any way to auto-approve a translation after X days so that a missing reviewer won't completely grind translations for a language to a halt?

I have searched issues and the web for this but wasn't able to find anything.

Thanks for your answers :)

**Comment by nijel** [comment, marked as answer]
Not right now, see https://github.com/WeblateOrg/weblate/issues/5447
